js/flightlog_parser.js

```diff
@@ -294,6 +295,7 @@ var FlightLogParser = function(logData) {
             motor_pwm_protocol        : "fast_pwm_protocol",
             pidsum_limit              : "pidSumLimit",
             pidsum_limit_yaw          : "pidSumLimitYaw",
+            rate_limits               : "rate_limits",
```

Comment:
This line is not needed. The translation table is only used when the variable name is different than the header name (usually happens with different versions of the firmware, they change the header name but in Blackbox we only have one var name and need to support all of them).

Reply:
Line removed in latest commit.